01-path-basics/103-kubernetes-concepts/readme.adoc

```diff
@@ -14,12 +14,12 @@ Kubernetes as a platform has a number of abstractions that map to API objects. T
 
 == Prerequisites
 
-This chapter uses a cluster with 3 master nodes and 5 worker nodes as described link:../102-your-first-cluster#multi-master[here].
+This chapter uses an EKS cluster with 3 worker nodes as described link:../102-your-first-cluster[here].
 
 All configuration files for this chapter are in the `01-path-basics/103-kubernetes-concepts/templates` directory.
 Please be sure to `cd` into that directory before running the commands below.
 
-    $ cd 01-path-basics/103-kubernetes-concepts/templates
+    $ cd ~/environment/aws-workshop-for-kubernetes/01-path-basics/103-kubernetes-concepts/templates
```

Comment:
Where is this `environment` directory coming from? I prefer the previous naming scheme since all directory names are relative to repo, w/o any reference to specific environment on a developer's laptop.